openpype/pipeline/tempdir.py

```diff
@@ -0,0 +1,57 @@
+"""
+Temporary folder operations
+"""
+
+import os
+from openpype.lib import StringTemplate
+from openpype.pipeline import Anatomy
+
+
+def create_custom_tempdir(anatomy=None):
```

Comment:
Project name should be requirement. It is the minimum required data for this function to work.